multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/FileStorageConfiguration.java

```diff
@@ -29,19 +28,13 @@ public ObjectStoreFileStorageFactoryBean objectStoreFileStorage(EnvironmentServi
 
     @Bean(name = "filterMultipartResolver")
     public MultipartResolver multipartResolver() {
-        CommonsMultipartResolver resolver = new CommonsMultipartResolver();
-        resolver.setMaxUploadSize(MAX_UPLOAD_SIZE);
-        resolver.setMaxUploadSizePerFile(MAX_UPLOAD_SIZE);
-        return resolver;
+        return new StandardServletMultipartResolver();
     }
 
     @Inject
     @Bean(name = "asyncFileUploadExecutor")
     public ExecutorService asyncFileUploadExecutor(ApplicationConfiguration configuration) {
-        return new ThreadPoolExecutor(5,
-                                      configuration.getFilesAsyncUploadExecutorMaxThreads(),
-                                      30,
-                                      TimeUnit.SECONDS,
+        return new ThreadPoolExecutor(5, configuration.getFilesAsyncUploadExecutorMaxThreads(), 30, TimeUnit.SECONDS,
```

Comment:
is this formatted?

Reply:
Yes, it was automatically done by the formatter.